### PR TITLE
es6-arrow-functions - Ray Le

### PR DESCRIPTION
**Pull request by lol-ray**
UhTxQm2E8P

### CODE REVIEW

File: es6-arrow-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    $jokeContainer.append($phrase);
+  }
+};
+
+const flash = {
+  laughingUrl: 'images/flash-laugh.gif',
+  laugh: function () {
+    setTimeout(function () {
+      $flashImage.setAttribute('src', this.laughingUrl);
+    }, 3000);
```
Comment (scbowler) [inline es6-arrow-functions/main.js:36]: This should be an arrow function here.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one update needed. Let me know if you have any questions.

**Comment by lol-ray** [comment]
image

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**